# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/dtlsidentity.h
# Commit: f18e61be455d
# Full Hash: f18e61be455d493f10a2cc651cd254859894e02e
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-26 03:52:35
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r 8136a17fbec9 -r f18e61be455d media/mtransport/dtlsidentity.h
--- a/media/mtransport/dtlsidentity.h	Mon Feb 25 21:48:20 2019 +0000
+++ b/media/mtransport/dtlsidentity.h	Mon Feb 25 21:50:42 2019 +0000
@@ -25,6 +25,7 @@
 class DtlsDigest {
  public:
   const static size_t kMaxDtlsDigestLength = HASH_LENGTH_MAX;
+  DtlsDigest() = default;
   explicit DtlsDigest(const std::string& algorithm) : algorithm_(algorithm) {}
   DtlsDigest(const std::string& algorithm, const std::vector<uint8_t>& value)
       : algorithm_(algorithm), value_(value) {